src/Parser/DeathKnight/Blood/Modules/Talents/RapidDecomposition.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class RapidDecomposition extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bpDamage = 0;
+  dndDamage = 0;
+  totalDamage = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.RAPID_DECOMPOSITION_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.BLOOD_PLAGUE.id && spellId !== SPELLS.DEATH_AND_DECAY_DAMAGE_TICK.id) {
+      return;
+    }
+    if (spellId === SPELLS.BLOOD_PLAGUE.id) {
+      this.bpDamage += calculateEffectiveDamage(event, 0.15);
+    }else {
+      this.dndDamage += calculateEffectiveDamage(event, 0.15);
+    }
+    this.totalDamage = this.bpDamage + this.dndDamage;
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.RAPID_DECOMPOSITION_TALENT.id} />}
+        value={`${this.owner.formatItemDamageDone(this.totalDamage)}`}
+        label='Damage Increase'
```

Comment:
How about calling it Rapid Decomposition instead? I think the "Damage Increase" part is obvious enough from the fact that it is DPS. Same for the tooltips below, could remove the "Damage Increase" and maybe add "Breakdown:" in a line at the top.

Reply:
Changed it. Decided the world break down wasnt needed.